Lexis+.js

```diff
@@ -0,0 +1,267 @@
+{
+	"translatorID": "419638d9-9049-44ad-ba08-fa54ed24b5e6",
+	"label": "Lexis+",
+	"creator": "bfahrenfort",
+	"target": "^https://plus.lexis.*/",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2023-04-09 22:52:49"
+}
+
+/*
+  ***** BEGIN LICENSE BLOCK *****
+
+  Copyright © 2023 Brandon Fahrenfort
+
+  This file is part of Zotero.
+
+  Zotero is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Zotero is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+  ***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, _url) {
+	if (doc.title.match(/.*results.*/)) {
+		return "multiple";
+	}
+	else if (doc.title.match(/[a-zA-Z. ]+\s§\s\d+/)
```

Comment:
```suggestion
	else if (/[a-zA-Z. ]+\s§\s\d+/.test(doc.title))
```
and similarly elsewhere (`match()` returns all possible matches and is therefore significantly less efficient then `test()` or even `search()`